tests/resources/config.yaml

```diff
@@ -1,6 +1,25 @@
 node_lines:
 - node_line_name: retrieve_node_line
   nodes:
+    - node_type: query_expansion
+      strategy:
+        metrics: [retrieval_f1, retrieval_recall]
+        top_k: 50
+        retrieval_module:
```

Comment:
Can you change this to 'retrieval_modules'?? Because you can use multiple modules in here..